dht-node/src/config/devop.ts

```diff
@@ -0,0 +1,13 @@
+// ATTENTION: DO NOT PUT ANY SECRETS IN HERE (or the .env)
+import dotenv from 'dotenv'
+import { getDevOpEnvValue } from './tools'
+dotenv.config()
+
+const DEVOP = {
```

Comment:
whats the definition of devops? What does this mean?

Reply:
see below